vng_api_common/oas.py

```diff
@@ -1,8 +1,5 @@
 """
```

Comment:
I'm not 100% sure, but we might be able to remove this file, since the schema fetching is no longer used?

Reply:
Decided to keep it, as we still use some of the utils in openzaak tests

Reply:
`obj_has_shape` is even used in loose-fk validators in Open Zaak, but i'm not sure about schema fetcher